dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1838,4 +1838,19 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
         @ApiParam(value = "GitHub installation ID", required = true) @FormParam("installationId") String installationId) {
         return githubWebhookRelease(repository, username, gitReference, installationId);
     }
+
+    @POST
+    @Path("/github/delete")
+    @Timed
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @UnitOfWork
+    @RolesAllowed({ "curator", "admin" })
+    @Operation(description = "Handles the deletion of a branch on GitHub. Will delete all workflow versions that match.", security = @SecurityRequirement(name = "bearer"), responses = @ApiResponse(responseCode = "418", description = "This code tells AWS Lambda not to retry."))
```

Comment:
- We have a constant for "bearer" now.  
- I think `HttpStatus + ""` is preferred over `"418"`

Reply:
It will only delete at most one workflow version per workflow, but it could do it for multiple workflows. Don't know if that's worth capturing, but I was initially confused as to how it could delete multiple versions when we just added a constraint that ensures only one version can exist.  I can't think of better way to phrase it off-hand. :(

Reply:
@garyluu I can't find 418 in HttpStatus.

Reply:
:(  not even HttpServletResponse has it either

Reply:
Was confused too, maybe the description should be longer. 
Like "Handles the deletion of a branch on GitHub. Will delete all workflow versions that match in all workflows that share the same repository path prefix" ? Or all workflows that are the same when ignoring workflow name?